File: 8KYU/Expressions Matter/solution.cs
using System.Linq;

public class Kata
{
    public static int ExpressionsMatter(int a, int b, int c)
        => new[]{a * (b + c),
                 a * (b * c),
                 a * b * c,
                 a + b + c,
                 (a + b) * c,
                 (a * b) * c,
                 (a * b) + c}.Max();
}

File: 8KYU/Expressions Matter/solution.go
package kata

func ExpressionMatter(a int, b int, c int) int {
	var xs []int
	xs = append(xs, a*(b*c))
	xs = append(xs, a*(b+c))
	xs = append(xs, a*b*c)
	xs = append(xs, a+b+c)
	xs = append(xs, (a+b)*c)
	xs = append(xs, (a*b)+c)

	max := xs[0]

	for _, x := range xs {
		if x > max {
			max = x
		}
	}
	return max
}
